### PR TITLE
feat: added java instrumentation for elastic search queries

### PR DESCRIPTION
**Pull request by hsinghkalsi**
# Summary

A concise description of the changes being introduced. If you are submitting a PR for a
new quickstart please review the pre merge checklist section to validate this PR is ready for review.

## Pre merge checklist

- [x ] Did you check you NRQL syntax? - Does it work?
- [x ] Did you check your dashboard image quality? -  Do they look good?
- [x ] Did you check that your alerts actually work?
- [x ] Did you include an InstallPlan and Documentation reference?
- [x ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [x ] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: quickstarts/java/elasticsearch-query/config.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: elasticsearch-query
+title: Elasticsearch query Java Agent Extension
+description: |
+  # What is elasticsearch?
+  Elasticsearch is a search engine based on the Lucene library. It provides a distributed, multitenant-capable full-text search engine with an HTTP web interface and schema-free JSON documents.
```
Comment (jpvajda) [inline quickstarts/java/elasticsearch-query/config.yml:5]: You may want to specify in the description this is specific for java.

**Review by jpvajda — CHANGES_REQUESTED** [review]
I've removed the icon.svg as we don't currently use that for the logo files, also I noticed the logo.svg doesn't seem to contain a logo, so you may want to double check that file.

File: quickstarts/java/elasticsearch-query/config.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  Elasticsearch provides a full Query DSL (Domain Specific Language) based on JSON to define queries
+
+summary: |
+  Java agent extension to monitor elasticsearch queries as dataStore queries with New Relic. 
+  Once deployed elasticsearch queries begin to show up in the New Relic UI under Databases for the application using elasticsearch queries.
+
+icon: icon.svg
```
Comment (jpvajda) [inline quickstarts/java/elasticsearch-query/config.yml:12]:
```suggestion
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/java/elasticsearch-query/config.yml (hunk 1 above)
Comment (hsinghkalsi) [reply quickstarts/java/elasticsearch-query/config.yml:5]: Added reference to java and java agent , i fixed logo, it is rendering good in https://www.svgviewer.dev/ .

File: quickstarts/java/elasticsearch-query/logo.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg height="-59" viewBox="86.8 87.5 404.2 80" width="2500" xmlns="http://www.w3.org/2000/svg">
```
Comment (jpvajda) [inline quickstarts/java/elasticsearch-query/logo.svg:1]: This SVG doesn't render anything as far as I can tell can you check to ensure this is correct?

Comment (jpvajda) [reply quickstarts/java/elasticsearch-query/logo.svg:1]: You can see how it renders here: https://github.com/newrelic/newrelic-quickstarts/blob/b9bdf0797d39261bcbcfb636f1282467484d4785/quickstarts/java/elasticsearch-query/logo.svg

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/java/elasticsearch-query/logo.svg (hunk 3 above)
Comment (hsinghkalsi) [reply quickstarts/java/elasticsearch-query/logo.svg]: apologies for that, fixed this https://github.com/hsinghkalsi/newrelic-quickstarts/blob/f607851ecf97ce7d1885415cf3045874e14af762/quickstarts/java/elasticsearch-query/logo.svg

**Review by jpvajda — APPROVED** [review]

File: quickstarts/java/elasticsearch-query/logo.svg (hunk 3 above)
Comment (jpvajda) [reply quickstarts/java/elasticsearch-query/logo.svg]: Thanks!

**Pull request merged**